### ISSUE TITLE
Add a "beta" label to the beta channel's icon

### ISSUE THREAD

**Issue by hawkrives** [StoDevX/AAO-React-Native#1030]
Fastlane has an action to edit the icon to add a label stripe across one corner. 

We should just use that.

**Comment by drewvolz** [comment]
http://nsscreencast.com/episodes/213-icon-badging-with-fastlane

**Comment by hawkrives** [comment]
I think it's as simple as adding 

```ruby
badge(label: 'BETA')
```

to the "beta" lane. 

(Note: working from memory here.)

**Comment by drewvolz** [comment]
Found the docs. https://docs.fastlane.tools/actions/#badge

```ruby
badge(dark: true)
badge(alpha: true)
badge(custom: "/Users/xxx/Desktop/badge.png")
badge(shield: "Version-0.0.3-blue", no_badge: true)
```

**Comment by hawkrives** [comment]
Found the external place where it's from: https://github.com/HazAT/badge

<img width="384" alt="screen shot 2017-05-07 at 9 25 48 pm" src="https://cloud.githubusercontent.com/assets/464441/25788031/c50e384e-336b-11e7-8917-ba4443653eeb.png">

**Comment by hawkrives** [comment]
… so we can literally do 

```ruby
badge
```

**Comment by drewvolz** [comment]
Closed by #1041
